File: examples-sdk/typescript/801-trigger-cron-basic.ts
#!/usr/bin/env tsx
// 801-trigger-cron-basic — TypeScript SDK equivalent
// Compile: vil compile --from typescript --input 801-trigger-cron-basic.ts --release

import { VilPipeline, VilServer, ServiceProcess } from "vil-sdk";

const server = new VilServer("app", 8080);
server.compile();

File: examples-sdk/python/801-trigger-cron-basic.py
#!/usr/bin/env python3
"""801-trigger-cron-basic — Python SDK equivalent
Compile: vil compile --from python --input 801-trigger-cron-basic.py --release
"""
import os
from vil import VilPipeline, VilServer, ServiceProcess

server = VilServer("app", port=8080)
server.compile()
